### DISCUSSION TITLE
Motorized fader instead of rotary encoder

### DISCUSSION THREAD

**Discussion by probonopd** [probonopd/MiniDexed#161] [Ideas]
The DX7 had a single slider for data entry.
That had the disadvantage that the position of the slider did not always match the parameter value.

Right now we are using a rotary encoder.

I guess the ultimate solution would be to use a **motorized** fader. It would move itself to the correct position "magically".
Yes, those things exist: [ALPS RSA0N11M9A0K](https://tech.alpsalpine.com/prod/e/html/potentiometer/slidepotentiometers/rsn1m/rsn1m_common.html)

![image](https://user-images.githubusercontent.com/2480569/164976268-2f1cd22a-6d23-488c-b47b-1facfa24338a.png)

And apparently they can be controlled using an Arduino.

https://motorizedfader.wordpress.com/2016/03/16/motorized-fader-surface-controller/

Complete overkill, but would make for a nice eyecatcher I guess.

(No, I won't be implementing this. But you never know - perhaps _someone_ will.)

**Comment by craigyjp** [comment]
I don't see the point, a motorized fader is for recall, but as you only have one input device you would be using that to select the parameter to recall, so you would be fighting with it.

Unless you mean over midi and then as you edit each parameter the slider could move relatively.

Or am I missing something?

**Reply by probonopd** [reply]
Well, to make a selection, you'd still have a button. The motor would of course be switched off after having retrieved the initial value of a parameter. No fighting!
